x/process/client/rest/query.go

```diff
@@ -110,13 +109,13 @@ func queryHashHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
-		var req api.CreateProcessRequest
-		if err := cliCtx.Codec.UnmarshalJSON(data, &req); err != nil {
+		var msg types.MsgCreate
```

Comment:
I will change this to a dedicated structure like https://github.com/mesg-foundation/engine/pull/1754